SpatialGDK/Documentation/content/handover-between-server-workers.md

```diff
@@ -1,8 +1,6 @@
 # Actor property handover between SpatialOS servers
 ## Overview
 
-<%(TOC)%>
```

Comment:
Why did we remove this?

Reply:
Because it's not really needed. Very few sections on this page so it just clutters the view.

Reply:
As Yoda said, "Use your in-page ToC wisely, Luke."

Reply:
"Your in-page ToC, wisely you use."